doc/contributing/fpga/debugging_with_ila.md

```diff
@@ -75,9 +75,9 @@ assign probe0_presync = {
   tdo_oe_o
 };
 
-prim_flop_2sync #(
+prim_generic_flop_2sync #(
   .Width(SyncWidth)
-) u_prim_flop_2sync_probe0 (
+) u_prim_generic_flop_2sync_probe0 (
```

Comment:
These changes should be reverted - we always instantiate the wrapper file (prim_flop_2sync), not the tech specific one.

Reply:
@msfschaffner Wait a minute. So in this PR, we are removing the `prim_flop_2sync`. Should we instead add the `prim_flop_generic_2sync` and then instantiate that in `prim_flop_2sync`?

Reply:
So there is a single instantiation of `prim_flop_generic_2sync` inside `prim_flop_2sync `

Reply:
No, the `prim_flop_2sync` becomes an automatically generated wrapper, generated by the primgen script. This is similar to e.g. `prim_flop`. What we have to implement are the `prim_generic*`, `prim_xilinx_*` etc flavors and place them in the right folder so that the primgen script can discover them.

Reply:
I am thinking something like this: https://github.com/lowRISC/opentitan/pull/21157